src/main/java/gui/Main.java

```diff
@@ -58,6 +58,7 @@ public void start(final Stage primaryStage) {
         // Set the scene and show primaryStage
         primaryStage.setTitle("TrackScape");
         primaryStage.setScene(monitorScene.createMonitorScene(primaryStage, stylesheet));
+        primaryStage.setFullScreen(true);
```

Comment:
is priaryStage.setMaximized(true) niet mooier

Reply:
Already done